autoload/minpac/impl.vim

```diff
@@ -427,4 +427,87 @@ function! minpac#impl#clean(args) abort
   endif
 endfunction
 
+function! s:syntax()
+  syntax clear
+  syn match minpacDash /^-/
+  syn match minpacName /\(^- \)\@<=.*/ contains=minpacStatus
+  syn match minpacStatus /\(-.*\)\@<=-\s.*$/ contained
+  syn match minpacStar /^\s\*/ contained
+  syn match minpacCommit /^\s\*\s[0-9a-f]\{7,9} .*/ contains=minpacRelDate,minpacSha,minpacStar
+  syn match minpacSha /\(\s\*\s\)\@<=[0-9a-f]\{4,}/ contained
+  syn match minpacRelDate /([^)]*)$/ contained
+
+  hi def link minpacDash    Special
+  hi def link minpacStar    Boolean
+  hi def link minpacName    Function
+  hi def link minpacSha     Identifier
+  hi def link minpacRelDate Comment
+  hi def link minpacStatus  Constant
+endfunction
+
+function! minpac#impl#status()
+  let l:result = []
+  let l:update_ran = exists('s:installed_plugins')
+  for l:name in keys(g:minpac#pluglist)
+    let l:pluginfo = g:minpac#pluglist[l:name]
+    let l:dir = l:pluginfo.dir
+    let l:plugin = { 'name': l:name, 'lines': [], 'status': '' }
+
+    if !isdirectory(l:dir)
+      let l:plugin.status = 'Not installed'
+    else
+      let l:commits = systemlist([g:minpac#opt.git, '-C', l:dir, 'log',
```

Comment:
I don't want to use `systemlist()` (or `system()`), because it opens a command window on Windows.
Can you use `s:system()` instead?

Reply:
I just tried, but that won't return any results. I tried increasing sleep to 1000m in s:system, but still i get empty results back. Do you have idea why?

Reply:
My bad. It works. Pushed a fix.